editor/src/components/navigator/navigator-utils.ts

```diff
@@ -672,33 +678,37 @@ function getNavigatorRowsForTree(
   const condensedTree = condenseNavigatorTree(navigatorTree)
 
   function walkTree(entry: NavigatorTree, indentation: number): Array<NavigatorRow> {
+    function walkIfSubtreeVisible(e: NavigatorTree, i: number): Array<NavigatorRow> {
+      if (
+        filterVisible === 'visible-navigator-targets' &&
+        'subtreeHidden' in entry &&
```

Comment:
I missed this yesterday, I think this is a little too opaque, a helper function would be more readable imo

Reply:
fixed in 65ce1aed1